arangod/RocksDBEngine/RocksDBCollection.cpp

```diff
@@ -1934,17 +1934,17 @@ void RocksDBCollection::destroyCache() const {
 void RocksDBCollection::invalidateCacheEntry(RocksDBKey const& k) const {
   if (useCache()) {
     TRI_ASSERT(_cache != nullptr);
-    bool banished = false;
-    while (!banished) {
+    do {
```

Comment:
Why did you use `do while` instead of just `while`?

Reply:
* to get rid of the `banished` variable
* to make the intent more clear: we always want to execute `banish()` at least once

Reply:
You write do while true, so my question is why don't use just while true?

Reply:
Also I forgot to put NIT, it's not really important for me, so you could just resolve it

It just looks like you start with "do while some condition", and then removed condition, so it looks obsolete for me.
But in general it doesn't matter